pom.xml

```diff
@@ -403,7 +403,7 @@
           <configuration>
             <argLine>-Xms512m -Xmx3g -Ddrill.exec.http.enabled=false
               -Ddrill.exec.sys.store.provider.local.write=false
-              -Dorg.apache.drill.exec.server.Drillbit.system_options="org.apache.drill.exec.compile.ClassTransformer.scalar_replacement=on"
+              -Dorg.apache.drill.exec.server.Drillbit.system_options="org.apache.drill.exec.compile.ClassTransformer.scalar_replacement=try"
```

Comment:
This isn't the change we want. Scalar replacement should be "on" for tests but "try" for production. Your change is to move it to "try" for tests.